projects/packages/my-jetpack/src/class-initializer.php

```diff
@@ -39,6 +39,17 @@ public static function init() {
 			return;
 		}
 
+		// Edit Jetpack plugin actions link.
+		$jetpack_filename = Product::get_installed_plugin_filename( 'jetpack' );
```

Comment:
Would this work with `jetpack-dev` ?

Reply:
I think so, but it's better @leogermani confirm it.

Reply:
Got confused about the filename 👍